packages/react-storage/package.json

```
@@ -79,7 +79,7 @@
       "name": "FileUploader",
       "path": "dist/esm/index.mjs",
       "import": "{ FileUploader }",
-      "limit": "21.6 kB"
+      "limit": "21.7 kB"
```

We can reduce later:
```suggestion
     "limit": "25.00 kB"
```